assessment-api/ddl.sql

```diff
@@ -0,0 +1,227 @@
+( assessment_taxon_id        integer       primary key,
+  assessment_tool_id         integer       not null,
+  parent_assessment_taxon_id integer,
+  assessment_taxon_name      varchar(50),
+  permalink                  varchar(500),
+  assessment_taxon_descr     varchar(2000),
+  created_by                 varchar(2000) not null,
+  created_at                 timestamp     not null,
+  updated_by                 varchar(2000),
+  updated_at                 timestamp);
+ 
+-- Create table taxon_dimensions
+
+create table assessment_taxon_dimensions
+( assessment_taxon_dimension_id serial        primary key,
+  assessment_taxon_id           integer       not null,
+  dimension_name                varchar(30)   not null,
+  created_by                    varchar(2000) not null,
+  created_at                    timestamp     not null,
+  updated_by                    varchar(2000),
+  updated_at                    timestamp);
+
+
+-- Create table requirements (products & services)
+
+create table requirements
+( requirement_id    serial        primary key,
+  requirement_name  varchar(100)  not null,
+  requirement_descr varchar(2000) not null,
+  created_by        varchar(2000) not null,
+  created_at        timestamp     not null,
+  updated_by        varchar(2000),
+  updated_at        timestamp);
+
+ 
+-- Create table requirement_taxons
+
+create table requirement_taxons
+( requirement_taxon_id serial        primary key,
+  requirement_id       integer       not null,
+  assessment_taxon_id  integer       not null,
+  created_by           varchar(2000) not null,
+  created_at           timestamp     not null,
+  updated_by           varchar(2000),
+  updated_at           timestamp);
+
+  
+
+-- Create table lot_requirement_taxons
+
+create table lot_requirement_taxons
+( lot_requirement_taxon_id serial        primary key,
+  lot_id               integer       not null,
+  requirement_taxon_id integer       not null,
+  created_by           varchar(2000) not null,
+  created_at           timestamp     not null,
+  updated_by           varchar(2000),
+  updated_at           timestamp);
+
+
+-- Create table supplier_submissions 
+  
+create table supplier_submissions
+( supplier_submission_id        serial        primary key,
+  lot_requirement_taxon_id      integer       not null,
+  supplier_id                   integer       not null,
+  assessment_submission_type_id integer       not null,
+  submission_reference          varchar(30),
+  submission_value              integer,
+  created_by                    varchar(2000) not null,
+  created_at                    timestamp     not null,
+  updated_by                    varchar(2000),
+  updated_at                    timestamp);
+
+
+-- Create table Assessments
+
+create table assessments(
+  assessment_id         integer       primary key,
+  buyer_organisation_id integer       not null,
+  assesment_name        varchar(100)  not null,
+  assessment_descr      varchar(2000) not null,
```

Comment:
As discussed - do we want to remove unused fields? (assessment_name, assessment_description) (possibly also buyer_organisation_id)

Reply:
yes will do

Reply:
If we want to know which organisation performed the assessment then we need to record this. One, to make sure we only pull back the correct organisation's assessment. Two, it could be useful for MI/BI purposes.